lib/liquid/utils.rb

```diff
@@ -81,5 +81,15 @@ def self.to_date(obj)
     rescue ::ArgumentError
       nil
     end
+
+    def self.to_liquid_value(obj)
+      # Check if to_liquid_value responds
```

Comment:
This isn't a very helpful comment. How about:
```ruby
# Enable "obj" to represent itself as a primitive value like integer, string, or boolean
return obj.to_liquid_value if obj.respond_to?(:to_liquid_value)
# Otherwise return the object itself
obj
```